tests/units/units.cpp

```diff
@@ -1725,27 +1862,31 @@ TEST(Units, isEquivalentBaseUnitNotInModel)
     u2->setName("u2");
     u2->addUnit(libcellml::Units::StandardUnit::RADIAN, 0, 1.0, 1.0);
     u2->addUnit("u", 0, 1.0, 1.0);
+    // test should fail because base unit "u" is not present in u1, regardless of whether "apples" exists or not
```

Comment:
Remove comment. There is no test around that comment, so no idea what this refers to.

Reply:
I've updated it, thanks.